concordia/static/scss/action-app.scss

```diff
@@ -234,7 +234,7 @@ main {
 
     &[data-mode='review'] {
         #transcriber-column {
-            display: none;
```

Comment:
This appears to have drifted between 53b014da3a26a19b8350e91c829289413c90a767 and f1c961bcfb005cffd5125b5c7a0e54c80d2f78e6 and it wasn't immediately obvious to me from the commit message that it was intentional to have both columns displayed in review mode but not transcribe mode.

Reply:
The intent is to have display: flex set for both columns when in review mode but only display: flex for transcribe column in transcribe mode. Even with both columns in review mode set to display: flex, only one displays at a time.